### PR TITLE
Moving user analytics events out of disableAnalytics check

### PR DESCRIPTION
**Pull request by JakeBat**
# Problem
User Analytics events were being toggled off by disableAnalytics config

# Solution
Moved UserAnalytics event firing outside of disableAnalytics check

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated? - N/A
- [ ] Has the RELEASE_GUIDELINES been updated? - N/A
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes? - N/A
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes? - N/A
- [ ] Have any new automated tests been implemented or the existing ones changed? - N/A
- [ ] Have any new manual tests been written down or the existing ones changed? - N/A
- [ ] Have any new strings been translated or the existing ones changed? - N/A

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Updated Browser Compatibility section in `README.md` to better indicate IE11, Firefox support
 - Public: Update English copy text for error message shown when no document is in the cameras view
 - Public: The `useMultipleSelfieCapture` configuration option is now stable and enabled by default
+- Public: Moved `UserAnalytics` event firing outside of `disableAnalytics` config check
```
Comment (stefaniacardenas) [inline CHANGELOG.md:20]: This should be moved under the next-version header

Comment (JakeBat) [reply CHANGELOG.md]: Updated

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1031-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1031-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**